e2e/provision/README.md

```diff
@@ -118,14 +123,193 @@ connection with that setting).
    from the cluster. For example:
 
    ```
+   $ kubectl get clusters
+   NAME       PHASE         AGE     VERSION
+   edge01     Provisioned   3h40m   v1.26.3
+   edge02     Provisioned   3h42m   v1.26.3
+   edge03     Provisioned   3h42m   v1.26.3
+   edge04     Provisioned   3h40m   v1.26.3
    $ kubectl get secret edge01-kubeconfig -o jsonpath='{.data.value}' | base64 -d > edge01-kubeconfig
    $ kubectl --kubeconfig edge01-kubeconfig get ns
    NAME                           STATUS   AGE
-   config-management-monitoring   Active   5m25s
-   config-management-system       Active   5m25s
-   default                        Active   5m46s
-   kube-node-lease                Active   5m48s
-   kube-public                    Active   5m48s
-   kube-system                    Active   5m48s
+   config-management-monitoring   Active   3h35m
+   config-management-system       Active   3h35m
+   default                        Active   3h39m
+   kube-node-lease                Active   3h39m
+   kube-public                    Active   3h39m
+   kube-system                    Active   3h39m
    $
    ```
+
+   You should also check that the Kind cluster came up fully with `kubectl get
+   machinesets`. Sometimes they do not all come up; it's not clear why yet, but
+   is probably a resourcing issue on the VM. For example, in this case only
+   three clusters had machines come up:
+
+   ```
+   $ kubectl get machinesets
+   NAME                                   CLUSTER    REPLICAS   READY   AVAILABLE   AGE     VERSION
+   edge01-md-0-6fk8w-9dc5bb56dxnpz69      edge01     3                              3h46m   v1.26.3
+   edge02-md-0-blqfh-57fc564884xjf865     edge02     3          3       3           3h49m   v1.26.3
+   edge03-md-0-8bss6-5b95ddf7b8xf8fg5     edge03     3          3       3           3h48m   v1.26.3
+   edge04-md-0-z7b8p-77748bffbfxwsjbc     edge04     3                              3h46m   v1.26.3
+   test01-md-0-2zkcp-666fb857c6x6jnzk     test01     3          3       3           3h55m   v1.26.3
```

Comment:
Where this test01 machineset came from?

Reply:
Just from other testing. I can remove it from the list to avoid confusion.

Reply:
Ok, I tweaked the responses of the command to be more in line with what you would see if you did this in the order of the README. I did it in a different order and so some things show up when they shouldn't. Eventually we can recapture command output on a clean run.